src/components/TodoItem/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './TodoItem.css';
+
+const TodoItem = ({ title, user, id, completed, handleCompleted }) => {
+  const todoItemClass = classNames('todo-item ', { completed });
```

Comment:
```suggestion
  const todoItemClass = classNames('todo-item', { completed });
```
there is no need to add spaces manually